src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData.Web/README.md

```diff
@@ -1,6 +1,6 @@
 # AI Chat with Custom Data
 
-This project is an AI chat application that demonstrates how to chat with custom data using an AI language model. Please note that this template is currently in an early preview stage. If you have feedback, please let us know!
+This project is an AI chat application that demonstrates how to chat with custom data using an AI language model. Please note that this template is currently in an early preview stage. If you have feedback, please take our [brief survey](https://aka.ms/dotnet-chat-template-survey).
```

Comment:
This link doesn't yet work. When will it? We're plastering the link in multiple places... are we happy with the survey?

Reply:
I tagged @jmatthiesen in Teams to make sure he is assigned to publish this survey and get the link working. It's the same link as in the app.